packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -11,8 +11,8 @@ import getRoutesConfig from 'App/Constants/routes-config';
 import { getAllowedLanguages, currentLanguage, changeLanguage } from 'Utils/Language';
 import LiveChat from '../../Elements/live-chat.jsx';
 
-const MenuLink = ({ link_to, icon, is_active, is_disabled, is_external, suffix_icon, text, onClickLink }) => {
-    if (is_external) {
+const MenuLink = ({ link_to, icon, is_active, is_disabled, is_language, suffix_icon, text, onClickLink }) => {
```

Comment:
I can't see any `is_language` prop being passed to the MenuLink component?

Reply:
It is currently passed to it on line 213.

Reply:
ah there it is 👍